packages/ember-htmlbars/tests/integration/attrs_lookup_test.js

```diff
@@ -1,154 +0,0 @@
-import compile from 'ember-template-compiler/system/compile';
```

Comment:
no worries, i rebased and it looks like the components folder was symlinked to the ember-htmlbars package so this is no longer an issue. Essentially the original test file was in `test/integration` while the new one is in `test/integration/components` so the relative import paths was failing for one but passing for the other.